packages/bundle-size/src/commands/uploadReport.js

```diff
@@ -0,0 +1,139 @@
+const { odata, TableClient, TablesSharedKeyCredential, TableTransaction } = require('@azure/data-tables');
+const chalk = require('chalk');
+
+const collectLocalReport = require('../utils/collectLocalReport');
+const { hrToSeconds } = require('../utils/helpers');
+
+/** @typedef {import('../utils/collectLocalReport').BundleSizeReportEntry} BundleSizeReportEntry */
+
+const AZURE_STORAGE_ACCOUNT = 'fluentbundlesize';
+const AZURE_STORAGE_TABLE_NAME = 'latest';
+const AZURE_ACCOUNT_KEY = process.env.BUNDLESIZE_ACCOUNT_KEY;
```

Comment:
there are various `process.env` refs within this package already. 

can we aggregate all of them within `environment.js` file  ?

```
const environment  = {
  AZURE_ACCOUNT_KEY: process.env.BUNDLESIZE_ACCOUNT_KEY,
  isCI: process.env.TF_BUILD,
  ...
  ...
}

module.exports = environment
```

Reply:
For `isCI`:

> As we plan we to publish this tool to run it in other repos (Tabster will be the first place to dogfood) I decided to use ci-info to avoid proprietary environment variables. Done in c6ed110.

https://github.com/microsoft/fluentui/pull/18322#discussion_r657753271

So this removes need in handling this env variable 😅

---

For `AZURE_ACCOUNT_KEY` and other `AZURE_*` constants: I placed them intentionally in this file to avoid leaking to other files as we plan to refactor upload part and it will have completely different constants. Do you think that there is a need to perform changes there before we will apply final changes?

Reply:
> https://github.com/watson/ci-info

interesting. How would this allow me to try out CI on local, I can just any existing CI env variable to true ?

> For AZURE_ACCOUNT_KEY and other AZURE_* constants: I

thanks for context. in that case I'm ok to keep it as is. in future we would embrace standard env variables processing via dotenv and such

Reply:
> > https://github.com/watson/ci-info
> 
> interesting. How would this allow me to try out CI on local, I can just any existing CI env variable to true ?

Yeah, it's determined based on https://github.com/watson/ci-info/blob/master/vendors.json (`env` key contains CI variable)

```bash
$ node -e "console.log(require('ci-info').isCI)"
false
```
```bash
$ CIRCLECI=true node -e "console.log(require('ci-info').isCI)"
true
```